api-admin/src/main/resources/api/openapi.yml

```diff
@@ -1143,43 +1143,22 @@ paths:
           content: { application/json: { schema: { type: object } } }
         '500':
           $ref: '#/components/responses/ServerError'
-  /api/adminUser/v1/{id}:
+  /api/adminUsers/v1/{id}:
     get:
       summary: Gets an admin user
       tags: [ admin_user ]
       operationId: get
       parameters:
         - { name: id, in: path, required: true, schema: { type: string, format: uuid } }
+        - { name: portalShortcode, in: query, required: false, schema: { type: string } }
       responses:
         '200':
-          description: admin user object
-          content: { application/json: { schema: { $ref: '#/components/schemas/AdminUserDto' } } }
```

Comment:
This was the only use of `AdminUserDto` in the spec, so that can be removed now too